src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.13.0-preview</Version>
+		<Version>1.13.1-preview</Version>
 		<PackageTags>Microsoft Azure SQL Management;SQL;SQL Management;</PackageTags>
 		<PackageReleaseNotes>
```

Comment:
Please delete the current release notes and add new release notes:
```
- Added support for Managed Instances, Managed Instance databases, and Managed Instance failover groups.
```

Reply:
Thanks Jared. I have update the release note now.